docs/source/RP3.rst

```diff
@@ -0,0 +1,210 @@
+
+Press and hold the [EDL] button
+
+.. figure:: /image/RubikPi/RubikPiEDL.png
+    :scale: 30%
+    :alt: EDL button location
+
+While continuing to hold the [EDL] button, connect the power supply into [PWR] USB port
+
+.. figure:: /image/RubikPi/RubikPiPWR.png
+    :scale: 30%
+    :alt: PWR port location
+
+While continuing to hold the [EDL] button, insert the Type-C cable into [USB 3.1] port and connect to your host computer.
+
+.. figure:: /image/RubikPi/RubikPiUSB.png
+    :scale: 30%
+    :alt: USB 3.1 port location
+
+Wait three seconds to enter 9008 mode, then release [EDL] button
+
+Flashing the board
+-----------------
+
+The RUBIK Pi will be automatically detected by Qualcomm Launcher. Select the device and click Flash button to begin updating the Renesas USB firmware.
+
+After successful USB firmware flashing, a dialog box will appear. Once the Launcher is waiting for user action to put the device into EDL mode again,
+the dialog box will display exactly: *Waiting for EDL device*, as seen in the following screen:
+
+.. figure:: https://www.thundercomm.com/rubik-pi-3/en/assets/images/Renesas_USB_firmware_flashing_completed_R-9c4eee492f90429747c3460a84d717cf.png
+    :align: center
+    :alt: Qualcomm launcher awaiting EDL
+
+*Credits: Thundercomm*
+
+Without closing Qualcomm Launcher, disconnect the device and place it into EDL mode again.
+As soon as the device is placed into EDL mode, the flashing operation begins automatically. You will see the following screen:
+
+.. figure:: https://www.thundercomm.com/rubik-pi-3/en/assets/images/OS_image_flashing_Progress-5544bef5da0e1e364be431eac5ecaf5d.png
+    :align: center
+    :alt: Qualcomm launcher flash OS image
+*Credits: Thundercomm*
+
+After the OS image is successfully flashed, the following confirmation screen will appear.
+
+.. figure:: https://www.thundercomm.com/rubik-pi-3/en/assets/images/OS_image_flashing_completed-ff45bd2899b7b3e02bdb63f28a862f42.png
+    :align: center
+    :alt: Qualcomm launcher flash OS successful
+*Credits: Thundercomm*
+
+Configure the device
+-----------------
+
+Click Configure Login, the launcher will then prompt to setup UART connection.
+
+.. note::
+    If the MicroUSB COM port on the host system is currently in use by another tool (e.g., PuTTY, Tera Term), please close it before proceeding.
+
+Without disconnecting any cable from the device, connect your host computer to the device with a USB Micro-B cable at the [DEBUG] port
+
+.. figure:: /image/RubikPi/RubikPiUART.png
+    :scale: 30%
+    :alt: DEBUG port location
+
+Once the USB Micro-B cable is connected, a serial terminal will open. When asked, type in the default username and password:
+
+* Username: ubuntu
+* Password: ubuntu
+
+The terminal will ask you to reset the password immediately, type in the default password again when prompted, then set your password.
```

Comment:
Emphaize you have to type in the password, I tried copying and pasting from a password manager and the paste did not go through serial terminal